src/map/utils/mobutils.cpp

```diff
@@ -1527,6 +1527,11 @@ a mobs regen rate, magic defense, triple attack rate from a table instead of har
                 mobutils::InitializeMob(PMob);
             }
         }
+        else
+        {
+            ShowError("Unable to find entity with groupId: %d, zoneId: %d.", groupid, groupZoneId);
```

Comment:
This has been deployed on Horizon for a while right? This doesn't trigger noisily from other or custom content?

Reply:
Happy to remove this if not wanted